src/main/generated/options/soot/AntTask.java

```diff
@@ -223,10 +228,6 @@ public Path createprocess_dir() {
             return process_dir.createPath();
         }
   
-        public void setderive_java_version(boolean arg) {
```

Comment:
@mbenz89 Do we need this method?

Reply:
This is auto generated. You will just have to fix the sot_options.xml file and then some files will be generated. Try it :)

Reply:
But I dont see any difference from our default branch to develop branch